### DISCUSSION TITLE
Read string-array from metadata in module

### DISCUSSION THREAD

**Discussion by giorgosioak** [LSPosed/LSPosed#1927] [Q&A]
Hello, is there a way to access a Resource from Xposed code?

Lets say we have `string-array` resource:
```xml
  <string-array name="mylist">
    <item>com.giorgosioak.connection_tester</item>
  </string-array>
```
I defined it as meta-data in `AndroidManifest.xml`:
```xml
<meta-data
    android:name="xposedlist"
    android:resource="@array/mylist" />
```
My question: If i to access it from my xposed class, how could i do it?

I had to ask since i can't find the way to do it. Thank you in advance

**Comment by yujincheng08** [comment]
whose?

**Reply by giorgosioak** [reply]
Don't understand your answer. I want to read a meta-data value from code

**Reply by yujincheng08** [reply]
I mean whose metadata you wanna read? The module's or the hooked app's?

**Reply by giorgosioak** [reply]
My module's

**Reply by yujincheng08** [reply]
XModuleResources + getPackageArchiveInfo
